ajna/utils/views.py

```diff
@@ -265,3 +271,87 @@ def get(self, request, **kwargs):
             serializer = self.serializer_class(page, many=True)
             page = serializer.data
         return self.paginator.get_paginated_response(page, additional_data)
+
+
+log = logging.getLogger(__name__)
```

Comment:
This should be at the top of the file just under imports, but before any of the functions